tokio/src/coop.rs

```diff
@@ -21,37 +21,18 @@
 //!
 //! It may look harmless, but consider what happens under heavy load if the
 //! input stream is _always_ ready. If we spawn `drop_all`, the task will never
-//! yield, and will starve other tasks and resources on the same executor. With
-//! opt-in yield points, this problem is alleviated:
+//! yield, and will starve other tasks and resources on the same executor.
 //!
-//! ```ignore
```

Comment:
Can we move these doc comments to a regular comment so that they can potentially be brought back later?

Reply:
Sounds good; done